feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/ui/PreviewScreenComponents.kt

```diff
@@ -129,24 +127,14 @@ private const val TAG = "PreviewScreen"
 private const val BLINK_TIME = 100L
 
 @Composable
-fun ElapsedTimeText(
-    modifier: Modifier = Modifier,
-    videoRecordingState: VideoRecordingState,
-    elapsedNs: Long
-) {
-    AnimatedVisibility(
-        visible = (videoRecordingState is VideoRecordingState.Active),
-        enter = fadeIn(),
-        exit = fadeOut(animationSpec = tween(delayMillis = 1000))
-    ) {
-        Text(
-            modifier = modifier,
-            text = elapsedNs.nanoseconds.toComponents { minutes, seconds, _ ->
-                "%02d:%02d".format(minutes, seconds)
-            },
-            textAlign = TextAlign.Center
-        )
-    }
+fun ElapsedTimeText(modifier: Modifier = Modifier, elapsedTimeUiState: ElapsedTimeUiState.Enabled) {
+    val currentUiState = rememberUpdatedState(elapsedTimeUiState)
```

Comment:
You shouldn't need this. Changes to the `elapsedTimeUiState` argument will already cause a recomposition. You're not using it anywhere outside of composition (such as in a `LaunchedEffect`).